integration-test/2022-disputed-borders.py

```diff
@@ -74,16 +79,14 @@ def test_admin_level_3_state(self):
                 'boundary': 'administrative',
                 'place': 'state',
                 'source': 'openstreetmap.org',
-                'admin_level:XX': '2'
+                'admin_level:XX': '2',
+                'recognized_by': 'YY'
             }),
         )
 
-        self.assert_has_feature(
```

Comment:
I find these remaining admin_level = 3 tests confusing... Either update them to reflect the LUA changes, or take them out.

Reply:
I'll take them out.

Reply:
I originally left them in to verify we don’t export any admin_level 3 things, but I thought the logic was suspect then

Reply:
Fixed in 9506b97782500824588c5745c04dc520b202da17